### ISSUE TITLE
Do you provide the indicator rabbitmq_identity_info?

### ISSUE THREAD

**Issue by qiu-peng20** [kbudde/rabbitmq_exporter#284]
Hi，Do you provide the indicator rabbitmq_identity_info?
dashboard needs to use this indicator  😄
![image](https://user-images.githubusercontent.com/55682965/173336200-41002498-e581-4e75-bceb-87ca09d38950.png)

**Comment by kbudde** [comment]
No, it's not available. Even with this metric I don't expect the dashboard to work.

The dashboard is probably for the official exporter.
